docs/assets/images/square/table_of_square_icons.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<style>
+body {font-family: Lato,proxima-nova,Helvetica Neue,Arial,sans-serif; background: #FFFFFF; color: #404040;}
+h1   {color: blue;}
+p    {color: green;}
+</style>
+<body>
+
+<h2>This document generates the image used on our index page, specifically the "launch with" page.</h2>
+<table>
+  <tr>
+    <td>&#xA0;&#xA0;&#xA0;<img src="cavatica_square.png" width="50">&#xA0;&#xA0;&#xA0;</td>
+    <td>&#xA0;&#xA0;&#xA0;<img src="cgc_square.png" width="50">&#xA0;&#xA0;&#xA0;</td>
+    <td>&#xA0;&#xA0;&#xA0;<img src="dnanexus_square.png" width="50">&#xA0;&#xA0;&#xA0;</td>
+    <td>&#xA0;&#xA0;&#xA0;<img src="galaxy_square.png" width="50">&#xA0;&#xA0;&#xA0;</td>
+    <td>&#xA0;&#xA0;&#xA0;<img src="nextflowtower_square.png" width="50">&#xA0;&#xA0;&#xA0;</td>
```

Comment:
Where did you get this logo? I can't find it in a Google search?

Looks like a disc golf basket. :)

Reply:
The Nextflow Tower icons were the most ornery to get, including in earlier drafts that used their text-based logo...

The text-based one from an earlier draft of this PR is from a SVG I converted to PNG, although the conversion was, ahem, rough. It was also all-white so I used the fill tool to make it show up on at all on this markup.
<img width="861" alt="full" src="https://user-images.githubusercontent.com/27784612/154358297-31c130e5-4019-41b4-bb4b-c51ced49836b.png">

As for the little icon, when you go to tower.nf, the logo does _briefly_ appear before the page redirects. (Speaking of which, is there an easy way to block these kinds of redirects?) Additionally, before that redirect, the page has a favicon. If I recall correctly, I think I used the favicon, which was in SVG format. This time I was more careful when converting it to PNG. But I might have actually used the icon at the bottom, I'm not sure. Either way it's pretty small -- 45 x 45 or so -- so every other icon on the index page was scaled with the limitations of the nextflow one in mind.

<img width="1363" alt="Screen Shot 2022-02-22 at 12 46 43 PM" src="https://user-images.githubusercontent.com/27784612/155216527-0d9355b6-59a9-4b31-b24d-8c0047b6cf44.png">

Reply:
We're using a different icon on the launch with button. Is there a reason they are different (I don't know). @NatalieEO didn't you add the button?

Reply:
Natalie probably has more info on this, but my understanding is as follows... 

It seems that Nextflow Tower has two logos, one which is rectangular with an X and one that is a small circle with a gaslamp. I went with the gaslamp because the other one seems to only exist as a full word like this, and earlier on we decided that square logos would be better. (Okay, yeah, circles are not squares but you get my drift.)

#### nextflow rectangle logo
![0d6acfc1-dee8-4120-b383-e69014707a1a](https://user-images.githubusercontent.com/27784612/155387940-98a68c30-d65f-46f8-9cc3-449228c72213.png)

I had always presumed that our launch-with was just cropped to the middle of the green version of the full-word logo. The green version has a dark patch on the left side of the X, which is also present in our version. 

#### our launch with PNG
![nextflow-icon](https://user-images.githubusercontent.com/27784612/155387734-4970cce7-b0b5-468d-a298-8e1294c03d31.png)

#### cropped slide from a youtube video (emphasis added)
![maxresdefault](https://user-images.githubusercontent.com/27784612/155388566-231ec29f-0f37-416d-91b2-e6047acfda84.jpg)


For another example of the gaslamp version of the logo, [it's also on their docs.](https://help.tower.nf/21.12/)

#### gaslamp logo
![index](https://user-images.githubusercontent.com/27784612/155388025-25fa4c81-edb9-49a7-ad1b-9292dac0219a.png)

Reply:
I think we should use the same logo in both places, whichever one we pick.

Reply:
Sorry, I missed this. I did add the Nextflow logo, but I honestly remember where I got it from lol. I doubt I cropped it myself

Okay I googled Nextflow logo and I got it from here https://github.com/nextflow-io/trademark

Reply:
@coverbeck @NatalieEO When you click on the launch-with button on a nextflow workflow, the page you are taken to use them gaslamp logo. I agree with Charles that we need consistency, but I'm wondering if it's the launch-with that should change? I don't know which is actually more recognizable at this point.

Reply:
We could reach out to Nextflow and ask them.

Reply:
In general, I would suggest we always reach out to the partners and ask what logos we should be using.  Many, including UCSC, take branding very seriously.